roles/database/tasks/initialize-raw-schema.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: List all schemas in the fpsd database.
+  become: true
+  become_user: postgres
+  command: psql -c '\dn' fpsd
```

Comment:
Yup. For some reason the `postgres_user` module is not idempotent, which is why running and stroing the result of this task is necessary.